File: Matnas/Common/VolunteerAndFamily.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Common
{
    public class VolunteerAndFamily
    {
        public int Id { get; set; }
        public Family Family { get; set; }
        public Volunteer Volunteer { get; set; }
        public Category Category { get; set; }
        public string Comments { get; set; }
        public Nullable<System.DateTime> DateAdded { get; set; }

    }
}

File: rmatnas/src/app/Classes/VolunteerAndFamily.ts
import { Family } from './Family';
import { Volunteer } from './Volunteer';
import { Category } from './Category';

export class VolunteerAndFamily {
    Id: number;
    Family: Family;
    Volunteer: Volunteer;
    Category: Category;
    Comments: string;
    DateAdded: string;
    /**
     *
     */
    constructor(family: Family, myvolunteer: Volunteer, category: Category, comments: string, dateAdded: string) {
        this.Comments = comments;
        this.Category = category;
        this.Family = family;
        this.Volunteer = myvolunteer;
        this.DateAdded = dateAdded;

    }
}
